docs/getting_started/installation.md

```diff
@@ -21,69 +21,70 @@ Please make sure your system meets the following specs:
+                first, and then install numpy<2 and PyTorch via pip.
+                
+                If the default PyTorch version does not work for you due to a CUDA version compatibility issue, follow instructions
+                on [the PyTorch website](https://pytorch.org/get-started/locally/) to add the correct index option to the pip install
+                line to get a different pytorch version.
+
+                ```shell
+                conda create -n omnigibson python=3.10
+                conda activate omnigibson
+                pip install "numpy<2" torch torchvision torchaudio
+                ```
+
+        2. Install OmniGibson:
 
             <div class="grid" markdown>
 
             !!! note "Install from PyPI (source not editable)"
 
                 ```shell
-                pip install --no-cache-dir omnigibson[isaac]
+                pip install omnigibson
                 ```
 
             !!! example "Install from GitHub (source editable)"
 
                 ```shell
                 git clone https://github.com/StanfordVL/OmniGibson.git
                 cd OmniGibson
-                pip install --no-cache-dir -e .[isaac]
+                pip install -e .
                 ```
 
             </div>
 
-            If this step fails, we recommend trying the [source installation](#-install-from-source-linux--windows) method.
-
-        4. Run Isaac Sim to accept the EULA:
+            !!! note "Nightly build"
 
-            ```shell
-            isaacsim
-            ```
+                The main branch contains the stable version of **`OmniGibson`**. For our latest developed (yet not fully tested) features and bug fixes, please clone from the `og-develop` branch.
 
-            !!! important "EULA Acceptance"
-                It is necessary to accept the Omniverse License Agreement (EULA) in order to use Isaac Sim.
-                The first time `isaacsim` is imported, you will be prompted to accept the EULA:
+        3. Run the installation script to  install Isaac Sim as well as **`OmniGibson`** dataset and assets:
 
-                ```
-                By installing or using Omniverse Kit, I agree to the terms of NVIDIA OMNIVERSE LICENSE AGREEMENT (EULA)
-                in https://docs.omniverse.nvidia.com/platform/latest/common/NVIDIA_Omniverse_License_Agreement.html
-                Do you accept the EULA? (Yes/No)
-                ```
-
-                You must respond with 'Yes' to proceed. Once the EULA is accepted, it should not appear on subsequent Isaac Sim calls. If the EULA is not accepted, the execution will be terminated.
+            ```{.shell .annotate} 
+            python -m omnigibson.install # (1)!
+            ```
 
-                **You might get some error dialogs that are safe to ignore. After accepting the EULA, you can close Isaac Sim.**
+            1. You can apply additional flag `--no-install-datasets` to skip dataset install.
```

Comment:
what about the launcher option? or is that done explicitly in the other section?

Reply:
Yep explicitly in the other section.